mockk/common/src/test/kotlin/io/mockk/it/CoroutinesTest.kt

```diff
@@ -51,8 +50,33 @@ class CoroutinesTest {
         }
     }
 
+    /**
+     * See issue #48
+     */
+    @Test
+    fun mockPrivateCoroutineCall() {
+        val myClassSpy = spyk(MockPrivateSuspendCls())
```

Comment:
I have been porting mockk to kotlin 1.5.0, and when upgrading, this test passes only if the spy is declared as 

```
spyk<MockPrivateSuspendCls>(recordPrivateCalls = true)
```

Can you please change this? It is indeed how this test should be, so IMHO it's wrong for it to pass as it is :)